### ISSUE TITLE
Last Month Sprint

### ISSUE THREAD

**Issue by gomezzz** [esa/neuralg#21]
Goals (by priority)

- Module has to have good usability
- Provide as much functionality as possible (at least eig)
- Enable user training / finetune models
- Support complex eigval , not complex matrices (for now)
- Add SVD
- Make it fast / more accurate

# Requirements (by priority)
- Easily imported / installed
- Not crash :) (memory etc.)
- A bit more documentation
- Eig, also various matrix sizes
- Support non-sym matrices / complex eigval
- Suppport SVD
- Some small testing with users 
- Training code has to be somewhat accessible
- Safe-mode
- Minimal model size / maximal accuracy

# Steps (by priority)
- [x] Train for non-sym matrices / complex eigval, update error bounds etc. 
- [x] Support SVD
- [x] @cantordust Get it work for more matrix sizes (either through full conv or other)
- [x] Notebook for training / finetune a model (run_training is for training / inspecting all models for the module , not a strong user focus, so limited docs etc. , while this one has to have good explanations on: 1. How to load and finetune (e.g. one existing model and train a bit) 2. How to get it into the module and have it use it. (Might need a function for ModelHandler to provide a custom model via path or similar?) 
- [x] Add a method `set_safe_mode()` which runs some sanity checks on inputs and outputs (e.g. eigval > 1e16 , input NaN etc.)
- [x] Do a little trade-off study between accuracy / model complexity
- [x] Move repo to esa @gomezzz 
- [x] Add a readme similar to https://gist.github.com/naoki-sawada/d58e6b06299be3ccae5b4d0c93d0587d or https://github.com/esa/torchquad , important: Minimal usage example, also needs to make clear the supported inputs / outputs, and mention training distribution. And mention the notebooks
- [x] Get on pypi to be pip installable
- [x] Ask some people to discord if they could try out

# Optional
- Have a logo
